demos/cypress-shared/package.json

```diff
@@ -27,9 +27,9 @@
         "build": "tsc",
         "clean": "tsc --build --clean",
         "cypress:open:client-example": "CYPRESS_BASE_URL='http://0.0.0.0:9230' cypress open",
-        "cypress:run:client-example": "CYPRESS_BASE_URL='http://0.0.0.0:9230' cypress run",
+        "cypress:run:client-example": "CYPRESS_BASE_URL='http://0.0.0.0:9230' cypress run --spec 'cypress/e2e/captcha.cy.ts,cypress/e2e/correct.captcha.cy.ts,cypress/e2e/correct.captcha.signup.cy.ts'",
         "cypress:open:client-example:frictionless": "CYPRESS_BASE_URL='http://0.0.0.0:9230' cypress open --env default_page='/frictionless'",
-        "cypress:run:client-example:frictionless": "CYPRESS_BASE_URL='http://0.0.0.0:9230' cypress run --env default_page='/frictionless'",
+        "cypress:run:client-example:frictionless": "CYPRESS_BASE_URL='http://0.0.0.0:9230' cypress run --env default_page='/frictionless' --spec 'cypress/e2e/honeypot-enabled.cy.ts'",
```

Comment:
Create an additional command `cypress:run:client-example:frictionless:honeypot` and specify the PROSOPO_HONEYPOT var on the command line. Add the additional command to the github workflows.

Reply:
Okay, I will try for this